examples/multi-gpu-movielens/tf_trainer.py

```diff
@@ -84,8 +86,9 @@ def seed_fn():
     inputs[col] = tf.keras.Input(name=col, dtype=tf.int32, shape=(1,))
 # Note that we need two input tensors for multi-hot categorical features
 for col in CATEGORICAL_MH_COLUMNS:
-    inputs[col + "__values"] = tf.keras.Input(name=f"{col}__values", dtype=tf.int64, shape=(1,))
-    inputs[col + "__nnzs"] = tf.keras.Input(name=f"{col}__nnzs", dtype=tf.int64, shape=(1,))
+    inputs[col] = tf.keras.Input(name=f"{col}__values", dtype=tf.int64, shape=(1,)), tf.keras.Input(
+        name=f"{col}__nnzs", dtype=tf.int64, shape=(1,)
+    )
```

Comment:
I think this syntax makes a tuple, but maybe it would be clearer with explicit parens?